submissions/BogdanGapon/HTML & CSS popup/style.css

```diff
@@ -0,0 +1,235 @@
+*,
+*::after,
+*::before {
+  -webkit-box-sizing: border-box;
+          box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  color: var(--main-color);
+  margin: 0;
+  padding: 0;
+}
+
+h1,
+h2,
+h3,
+h4,
+ul,
+p {
+  margin: 0;
+  padding: 0;
+}
+
+img {
+  display: block;
+}
+
+li {
+  list-style: none;
+  padding: 0;
+}
+
+.section {
+  padding: 50px 0;
+}
+
+.container {
+  max-width: 2860px;
+  margin: 0 auto;
+}
+
+.header {
+  padding: 20px;
+  background-color: #fff;
+}
+
+.header-nav__list {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: end;
+      -ms-flex-pack: end;
+          justify-content: flex-end;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  gap: 25px;
+}
+
+.header-nav__item {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: center;
+      -ms-flex-pack: center;
+          justify-content: center;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  -ms-flex-preferred-size: 30px;
+      flex-basis: 30px;
+}
+
+.header-nav__link {
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: flex;
+  -webkit-box-pack: center;
+      -ms-flex-pack: center;
+          justify-content: center;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  text-decoration: none;
+  color: rgb(17, 15, 15);
+  -webkit-transition: 250ms color cubic-bezier(0.075, 0.82, 0.165, 1);
+  transition: 250ms color cubic-bezier(0.075, 0.82, 0.165, 1);
+}
+.header-nav__link:hover, .header-nav__link:focus {
+  color: rgb(170, 63, 63);
+}
+.header-nav__link.link-img:hover {
+  -webkit-box-shadow: 2px 2px 8px 2px rgba(18, 20, 22, 0.14);
+          box-shadow: 2px 2px 8px 2px rgba(18, 20, 22, 0.14);
+}
+
+.visually-hidden {
+  position: absolute;
+  -webkit-appearance: none;
+     -moz-appearance: none;
+          appearance: none;
+}
+
+.custom-checkbox {
+  position: relative;
+}
+.custom-checkbox::before {
+  content: " ";
+  display: -webkit-inline-box;
+  display: -ms-inline-flexbox;
+  display: inline-flex;
+  -webkit-box-pack: center;
+      -ms-flex-pack: center;
+          justify-content: center;
+  -webkit-box-align: center;
+      -ms-flex-align: center;
+          align-items: center;
+  width: 30px;
+  height: 30px;
+  background-image: url(../img/popup-button.png);
+  background-repeat: no-repeat;
+  background-size: 30px 30px;
+  cursor: pointer;
+}
+
+.popup-wrapper {
+  display: none;
+  position: absolute;
+  top: 65px;
+  right: 50px;
+  width: 350px;
+  height: 400px;
+  padding: 30px 15px 30px;
+  border: 1px solid rgba(128, 128, 128, 0.4);
+  -webkit-box-shadow: 1px 1px 2px rgba(128, 128, 128, 0.4);
+          box-shadow: 1px 1px 2px rgba(128, 128, 128, 0.4);
+  background-color: #fff;
```

Comment:
You should use css variables if value is used more then twice.
Please, fix such cases in all files.